src/foam/u2/dao/MDBrowserListView.js

```diff
@@ -0,0 +1,332 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.dao',
+  name: 'MDBrowserListView',
```

Comment:
Could we rename this MDDAOController?

Reply:
the others are MDUpdateBrowserView, MDCreateBrowserView. Should also be renamed?